template/src/store/mmkv/index.js

```diff
@@ -0,0 +1,18 @@
+import {MMKV} from 'react-native-mmkv'
+
+export const storage = new MMKV()
+
+export const mmkvStorage = {
+  setItem: (key, value) => {
+    storage.set(key, value)
+    return Promise.resolve(true)
```

Comment:
Because this is format type of localStorage in redux-persist lib. Its need return a promise.

  interface Storage {
    getItem(key: string, ...args: Array<any>): any;
    setItem(key: string, value: any, ...args: Array<any>): any;
    removeItem(key: string, ...args: Array<any>): any;
  }
  
    interface WebStorage extends Storage {
    /**
     * @desc Fetches key and returns item in a promise.
     */
    getItem(key: string): Promise<string | null>;
    /**
     * @desc Sets value for key and returns item in a promise.
     */
    setItem(key: string, item: string): Promise<void>;
    /**
     * @desc Removes value for key.
     */
    removeItem(key: string): Promise<void>;
  }